release/scan_docker_images.sh

```diff
@@ -2,6 +2,39 @@
 
 source ../_liferay_common.sh
 
+function check_usage {
+	if [ -z "${LIFERAY_IMAGE_NAMES}" ]
```

Comment:
@lucasmiranda0, we should also check the existence of the variables `LIFERAY_PRISMA_SECRET` and `LIFERAY_PRISMA_ACCESS_KEY`. :wink: Both should also be included in the `print_help`.